mypy/build.py

```diff
@@ -774,10 +785,14 @@ def random_string() -> str:
     return binascii.hexlify(os.urandom(8)).decode('ascii')
 
 
+def compute_md5_hash(text: str) -> str:
```

Comment:
This probably shouldn't have "md5" in its name -- just `compute_hash()` sounds better, in case we want to switch hash implementations in the future.